examples/pytorch/GATNE-T/src/main.py

```diff
@@ -0,0 +1,313 @@
+from collections import defaultdict
+import math
+import os
+import sys
+import time
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import tqdm
+from numpy import random
+from torch.nn.parameter import Parameter
+import dgl
+import dgl.function as fn
+
+from utils import *
+
+
+# Build graph, treat all nodes as the same type
+def get_graph(network_data, vocab):
```

Comment:
please provide docstring to explain what are the inputs, what is their type, etc.

Reply:
Thanks for your suggestion. Already add comments for it.